lib/apigateway.js

```diff
@@ -11,10 +11,24 @@ function sleep(s) {
 	});
 }
 
+/**
+ * Constructor method for APIGateway
+ * @param  {object}  options  - AWS sdk class options. Can contain region, profile and filename
```

Comment:
- Add a newline between the description of the method and the param list.
- Remove the dash (`-`) between `options` and the description.